srcpkgs/dkms/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'dkms'
 pkgname=dkms
-version=2.8.4
-revision=2
+version=3.0.6
+revision=1
 conf_files="/etc/dkms/framework.conf"
 depends="bash kmod gcc make coreutils"
```

Comment:
Please add a dependency on `xbps-triggers>=0.123_1` to force updates to pull the new xbps-triggers, otherwise the trigger will be broken should people attempt a partial update. After that, this is ready for merge.

Reply:
Done.